mysql/pom.xml

```diff
@@ -86,4 +86,98 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>release</id>
```

Comment:
The release profile is used by the Jenkins Pipeline scripts.  It's the same for all the fabric8 projects so it's easy to add things like the gpg signing etc.